### ISSUE TITLE
Lecture Notes Folder

### ISSUE THREAD

**Issue by jmbaumer** [kevinaangstadt/CS340-S23-Notes#170]
Do we have a consensus on what we want to do with the lecture notes folder?

**Comment by hnsengimana** [comment]
I think we should remove it and have all of the lecture notes in the main folder.

**Comment by jmbaumer** [comment]
@hnsengimana I agree. Is there anyone that disagrees with removing the lecture notes folder?

**Comment by pamaph20** [comment]
100% delete it because all that should be in this repo are lecture notes. However, we could make use of a folder just for the exam guides.
